combat-trainer.lic

```diff
@@ -2699,7 +2694,7 @@ class TrainerProcess
     DRC.bput("kiss #{@prayer_mat}", 'You bend forward to kiss')
 
     if DRCI.get_item?("wine", @theurgy_supply_container)
-      DRC.bput("pour wine on #{@prayer_mat}", 'You quietly pour', 'Pour what', 'You set aside')
+      DRC.bput("pour wine on #{@prayer_mat}", { 'timeout' => 2, 'suppress_no_match' => true }, 'You quietly pour', 'Pour what', 'You set aside')
```

Comment:
> Making the wine pour not match, and timeout faster because if the wine isn't blessed, there is no messaging of any kind, and always causes a 15 second hang.

Since this is unusual, could you add a comment explaining it?